AnagramSolver.Tests/WordRepositoryTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using AnagramSolver.BusinessLogic.Classes;
+using NUnit.Framework;
+using System;
+
+namespace AnagramSolver.Tests
+{
+    [TestFixture]
+    public class WordRepositoryTests
+    {
+        private WordRepository _wordRepository;
+        public const string _filePath = "C:\\Users\\jonas.jaugelis\\source\\repos\\AnagramSolver\\zodynas.txt";
+        [SetUp] public void Setup()
+        {
+            _wordRepository = new WordRepository();
+        }
+
+
+        [Test]
+        public void Should_ReturnListOfWords_When_GivenFileOfWords()
+        {
+            var allWords = _wordRepository.GetAllWords(_filePath, 4);
+            Assert.That(allWords, !Is.Null);
+        }
+        [Test]
+        [TestCase("C:\\Users\\jonas.jaugelis\\source\\repos\\AnagramSolver\\zodynas2.txt")]
```

Comment:
Discussed about TestCase in a meeting. Consider adding a variable path in Assert section of test.